nbs/examples/Getting_Started.ipynb

```diff
@@ -0,0 +1,435 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    # You can play with different hyparameters
</pre><p>Comments should be specific and clear. Something like: "Try different hyperparmeters to improve accuracy." </p><p> </p><p>Please comment the relevant lines</p><p><code></code><code>max_epochs,</p><p>scaler_type,</p><p>encoder_hidden,</p><p>decodder_hiden</code><code></code></p><p> </p><p>and </p><p><code>n_freq_downsample</code></p><p> </p><p> </p><p>Also, it might be good to separate in different steps the configuration of the models and the instantion. of the <code>NueralForecast</code> class.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/323/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>